.github/workflows/pr-deploy.yaml

```diff
@@ -410,7 +410,7 @@ jobs:
             --first-user-username coder \
             --first-user-email pr${{ env.PR_NUMBER }}@coder.com \
             --first-user-password $password \
-            --first-user-trial \
+            --first-user-trial=false \
```

Comment:
We are currently not making any use of a premium license in PR deployments. If needed, we can always upload one manually.